src/constants/endpoint.ts

```diff
@@ -1,2 +1,3 @@
 export const BASE_ENDPOINT = '/api';
 export const HEALTH_ENDPOINT = '/health';
+export const SWAGGER_ENDPOINT = '/swagger';
```

Comment:
new endpoint constant, used both at root, and inside of `/api` to redirect `api/docs` as a gesture of decency 🧐